portal/src/store/modules/collections.js

```diff
@@ -124,6 +124,24 @@ export default {
       } else {
         $log.error('Validate error: ', { id, params })
       }
+    },
+    async getNextMaterialInMyCollection({ commit }, { id, params }) {
```

Comment:
Deze lijkt echt precies hetzelfde te doen als die hierboven. Kunnen we die niet beter hergebruiken en afhankelijk van de `params.page` iets anders doen?

Reply:
helemaal gelijk heb je, het is mijn onzekerheid die hier een beetje opspeelt😊